mathics_scanner/generate/build_tables.py

```diff
@@ -293,6 +293,7 @@ def compile_tables(data: dict) -> dict:
     "data_dir", type=click.Path(readable=True), default=DEFAULT_DATA_DIR, required=False
 )
 def main(field, output, data_dir):
+    print(f"writting field,{field}, into ",output)
```

Comment:
Stray print? (And "writing" is misspelled)

Reply:
I put this temporarily to see where the table was created. Accidentally was pushed here. It is gone now.